core-bundle/contao/languages/en/tl_preview_link.xlf

```diff
@@ -50,6 +50,9 @@
       <trans-unit id="tl_preview_link.share.1">
         <source>Copy the preview link ID %s to your clipboard.</source>
       </trans-unit>
+      <trans-unit id="tl_preview_link.copied_message">
+        <source>The link was copied to the clipboard.</source>
```

Comment:
```suggestion
        <source>Copied to clipboard!</source>
```

The default sounds way better, don't you think?

Reply:
If we go that route, the key should probably be in `MSC` then.

Reply:
See 8d57652b51c1a6660a8ac0d08af33171f993d7d1